main.tex

```diff
@@ -59,37 +60,36 @@
 %
 Here we use Illumina whole-genome sequence data from phase 1 of the \emph{Anopheles gambiae} 1000 Genomes Project (Ag1000G) to provide a comprehensive account of genetic variation in the \emph{Vgsc} gene in mosquito populations from 8 African countries.
 %
 In addition to the three known resistance alleles, we describe 20 non-synonymous nucleotide substitutions at appreciable frequency in one or more populations that are previously unknown in \textit{Anopheles} mosquitoes.
 %
-Thirteen of these novel alleles were found to occur almost exclusively on haplotypes carrying the known \texttt{L995F} resistance allele, and may be secondary mutations which enhance or compensate for the \texttt{L995F} resistance phenotype.
+Thirteen of these novel alleles were found to occur almost exclusively on haplotypes carrying the known \texttt{L995F} resistance allele, and may enhance or compensate for the \texttt{L995F} resistance phenotype.
 %
-A novel \texttt{I1527T} substitution, which is adjacent to a predicted pyrethroid binding site in the channel molecule, was found in tight linkage with either of two alleles causing a \texttt{V402L} substitution, similar to a combination of substitutions found to cause pyrethroid resistance in several other insect species.
+A novel mutation \texttt{I1527T}, which is adjacent to a predicted pyrethroid binding site, was found in tight linkage with either of two alleles causing a \texttt{V402L} substitution, similar to a combination of substitutions found to cause pyrethroid resistance in several other insect species.
 %
-We analyse the genetic backgrounds on which these non-synonymous alleles are found, to determine which alleles have experienced recent positive selection, and to refine our understanding of the spread of resistance between species and geographical locations.
+We analyse the genetic backgrounds on which non-synonymous alleles are found, to determine which alleles have experienced recent positive selection, and to refine our understanding of the spread of resistance between species and geographical locations.
 %
 We describe twelve distinct haplotype clusters with evidence of recent positive selection, five of which carry the known \texttt{L995F} resistance allele, five of which carry the known \texttt{L995S} resistance allele, one of which carries the novel \texttt{I1527T} allele, and one of which carries a novel \texttt{M490I} allele.
 %
-Seven of these clusters are localised to a single geographical location, and five comprise haplotypes from two or more countries, providing new information about the geographical distribution and spread of resistance.
+Seven of these clusters are localised to a single geographical location, and five comprise haplotypes from different countries, in one case separated by over 3000km, providing new information about the geographical distribution and spread of resistance.
```

Comment:
Technically I think you're supposed to have a space between numbers and units, i.e., "3000 km".